yjit/src/asm/mod.rs

```diff
@@ -196,6 +196,9 @@ impl CodeBlock
         // doing bad accesses with it.
         assert!(pos < self.mem_size);
         self.write_pos = pos;
+
+        // Clear the error flag to allow checking for new errors.
+        self.dropped_bytes = false;
```

Comment:
Hmm yeah this does seem kind of risky. It's for this assert to make sure that patching during invalidation actually wrote out bytes. 

```
assert!(!cb.has_dropped_bytes(), "patches should have space and jump offsets should fit in JMP rel32");
```

I thought I would reset the error because global invalidation should be able to patch code even after OOM. I suppose I could remove the assert for now.